lighthouse-core/lib/i18n.js

```diff
@@ -251,5 +274,7 @@ module.exports = {
   getDefaultLocale,
   getRendererFormattedStrings,
   createMessageInstanceIdFn,
+  getDisplayValue,
+  resolveIcuMessageInstanceId,
```

Comment:
does this need to be exported and public? could keep it _private to the file.

Reply:
yeah sure if you'd prefer :)